logstash-core/build.gradle

```diff
@@ -151,10 +151,9 @@ idea {
     }
 }
 
-def customJRubyDir = project.hasProperty("custom.jruby.path") ? project.property("custom.jruby.path") : ""
```

Comment:
*custom.jruby.path* would need more work to get right, 
should be unnecessary as we only need to replace *vendor/jruby* and no longer the complete.jar

Reply:
> custom.jruby.path would need more work to get right,

was wrong here - the `custom.jruby.path` bootstrap mechanism in the base `build.gradle` simply skips downloading JRuby and will (build and) copy a `vendor/jruby` from the locally build jruby artifact. there's no special need to select the proper *jruby-complete.jar* from the custom location since the *logstash-core* now relies on the `vendor/jruby/lib/jruby.jar` as an api dependency.

thus actually in case of a `./gradlew clean test -Pcustom.jruby.path=...` everything operates as before